Source/cielim/Public/CelestialBody.h

```diff
@@ -0,0 +1,44 @@
+// Fill out your copyright notice in the Description page of Project Settings.
```

Comment:
:small_red_triangle_down: Remove this text.

Reply:
Fixed